File: C Programming/Competitive Programming/Coding Challenges 6/program2.c
// 2. Accept single digit number from user and print it into word.
// Input : 9
// Output : Nine
// Input : -3
// Output : Three
// Input : 12
// Output : Invalid Number

#include<stdio.h>
void Display(int iNo)
{
    char *arr[] ={"Zero","One","Two","Three","Four","Five","Six","Seven","Eight","Nine"};


    if (iNo<0)
    {
        iNo=-iNo;
    }
    printf("Output is : ");

    if (iNo >= 0 && iNo <= 9)
    {
        printf("%s",arr[iNo]);
    }
    
    else
    {
        printf("Invalid Number");
    }
    
}

int main()
{
    int iValue = 0;

    printf("Enter number: ");
    scanf("%d",&iValue);

    Display(iValue);

    return 0;
}

File: Java Programming/Competitive Programming/Coding Challenges 12/program2.java
// 2. Accept single digit number from user and print it into word.
// Input : 9
// Output : Nine
// Input : -3
// Output : Three
// Input : 12
// Output : Invalid Number

import java.util.Scanner;

class program2
{
    static void Display(int iNo)
    {
        String[] Arr = {
            "Zero", "One", "Two", "Three", "Four",
            "Five", "Six", "Seven", "Eight", "Nine"
        };

        if (iNo < 0)
        {
            iNo = -iNo;
        }

        System.out.print("Output is : ");

        if (iNo < Arr.length)
        {
            System.out.print(Arr[iNo]);
        }
        else
        {
            System.out.print("Invalid Number");
        }
    }

    public static void main(String[] args)
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0;

        System.out.print("Enter number: ");
        iValue = sobj.nextInt();

        Display(iValue);

        sobj.close();
    }
}